lib/signal_reader/signal_reader.c

```diff
@@ -74,7 +74,7 @@ SignalReader* signal_reader_alloc(const GpioPin* gpio_pin, uint32_t size) {
     SignalReader* instance = malloc(sizeof(SignalReader));
 
     instance->pin = gpio_pin;
-    instance->pull = GpioPullNo;
+    instance->pull = GpioPullDown;
```

Comment:
Is this needed as it looks like `iso15693_parser_alloc` is calling `signal_reader_set_pull()` already.

If you can receive any 15693 signal from the reader then it's working, not having the pull-down set results in complete deafness.

Reply:
You are right, my mistake. Reverted this change.

I can successfully emulate iso15693, slix and picopass tags using proxmark as a reader. But proxmark use OOK modulation for these protocols, and I wonder how flipper emulation works with readers which use ASK modulation

Reply:
If you're running an up-to-date version (I recommend running master not the release) of the iceman fork - I added a --shallow option to `hf iclass` commands that will make it use ASK modulation instead of OOK which is better than nothing if you don't have a HID reader that does ASK.

Reply:
Testes with --shallow option - works fine. Thanks!